### PR TITLE
Split release creation into own task

### PR DESCRIPTION
**Pull request by MarcelMue**
This splits release creation into its own task and makes it composable. Tested with cncf / aws tests and works fine.

### CODE REVIEW

File: tekton/tasks/create-cluster.yaml [hunk 1]
```diff
@@ -3,11 +3,9 @@ kind: Task
 metadata:
   name: create-cluster
   namespace: test-workloads
+  labels:
+    app.kubernetes.io/name: "standup"
```
Comment (tfussell) [inline tekton/tasks/create-cluster.yaml:7]: Wasn't this for selecting tasks to kustomize the standup image version? Are you going to do that in a separate PR?

Comment (MarcelMue) [reply tekton/tasks/create-cluster.yaml:7]: I will try again in a separate PR - I can delete if you want it gone for now.

Just need some time to work on it.

Comment (tfussell) [reply tekton/tasks/create-cluster.yaml:7]: Eh, doesn't hurt anything.

**Review by tfussell — APPROVED** [review]
